### PR TITLE
Aerogear 3576 postgres

### PR DESCRIPTION
**Pull request by cfoskin**
Postgres integration - includes the creation of the tables when the app starts and updating the resolver to retrieve from postgres instead of the hard coded values.

### CODE REVIEW

**Comment by cfoskin** [comment]
@josemigallas I just need to resolve the lint errors

File: package.json [hunk 1]
```diff
@@ -24,11 +24,15 @@
     "express": "^4.16.3",
     "express-graphql": "^0.6.12",
     "graphql": "^0.13.2",
+    "graphql-relay": "^0.5.5",
+    "graphql-sequelize": "^8.3.1",
```
Comment (pb82) [inline package.json]: i think we can remove those two

**Comment by cfoskin** [comment]
@pb82 @josemigallas  Tests are failing due to sequelize relying on a running postgres

File: package.json (hunk 1 above)
Comment (cfoskin) [reply package.json]: 👍

**Comment by pb82** [comment]
@cfoskin Ok, i can take a look at that and make the tests work without a database.

File: server/models/dataSource.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+module.exports = (sequelize, DataTypes) => {
```
Comment (josemigallas) [inline server/models/dataSource.js]: Could we stick with ES6 syntax? For example `export default function getDataSource() { ... }`.

File: server/models/index.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    });
+} else {
+    db = new Sequelize(config.database, config.username, config.password, {
+        host: config.host,
+        dialect: "postgres"
+    });
+}
+
+const database = db;
+const dataSource = require("./dataSource")(database, Sequelize);
```
Comment (josemigallas) [inline server/models/index.js]: Not a fan of this syntax 😅 could we import `getDataSource` on the top instead?

File: server/server.js [hunk 4]
```diff
@@ -6,6 +6,7 @@ import setupGraphQLServer from "./gql";
 
 const App = express();
 let server = null;
+const database = require("./models");
```
Comment (josemigallas) [inline server/server.js]: Let's use `import` instead.

**Pull request merged**